content/docs/user-guide/action-manager/fundamentals/concepts/actions.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Actions
+linkTitle: Actions
+description: 
+weight: 101
+---
+
+The Action Manager system provides interfaces to register actions and related frameworks to interact with the O3DE Editor.
+
+
+## Action Context
+
+An Action Context is an object that owns a collection of Actions.
+
+An Action Context is comprised of:
+
+* A string **identifier** that makes it possible to reference it in API calls;
+* A **name** string, which is meant to be a human-readable description to show in UI (currently unused);
+
+It is possible to register a new Action Context via the following API call:
+
+```
+
+AzToolsFramework::ActionContextProperties contextProperties;
+contextProperties.m_name = "Test Action Context";
+
+QWidget* widget = new QWidget();
```

Comment:
This widget is unused here, we probably need to update this to show the separate `AssignWidgetToActionContext` that was added.

Reply:
I'll just remove it from here and add a section in the hotkey page.